Tests/Blazorise.E2E.Tests/ReadMe.md

```diff
@@ -0,0 +1,39 @@
+# Playwright E2E Test Project
+
+This test project uses Playwright with NUnit.
+NUnit was chosen since there is already an helpful playwright helper package for NUnit.
+
+Below you will find instructions on how to use Playwright. This assumes you will be using the command line and that you will be positioned in the root of the E2E tests project.
+
+## Install
+- Build the project, either by using the dotnet build command or Visual Studio.
+- Execute the playwright powershell script to install necessary dependencies (browser dependencies, etc...), i.e: `powershell .\bin\Debug\net7.0\playwright.ps1 install`
```

Comment:
Can this be done automatically without needing to run powershell?

Reply:
Well, this is a one time thing to initially install the dependencies, but maybe you can add a build post step in the csproj, no idea if those are still a thing in dotnet core.

Reply:
But I can see a problem on having it run automatically. If it throws an error running the script (i.e needs admin or policy modification) then the user might be puzzled to what's happening.

Reply:
Don't think it's worth chasing this. let me know if you agree.

Reply:
Yes, leave it for now.